### PR TITLE
Create PULL_REQUEST_TEMPLATE.md

### PR DESCRIPTION
**Pull request by shivakaushal**
Ref: sendgrid/sendgrid-nodejs#543

---
This change is [Reviewable](https://reviewable.io/reviews/sendgrid/sendgrid-nodejs/547)

### CODE REVIEW

**Comment by SendGridDX** [comment]
[CLA assistant check](https://cla.sendgrid.com/sendgrid/sendgrid-nodejs?pullRequest=547) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.sendgrid.com/sendgrid/sendgrid-nodejs?pullRequest=547) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.sendgrid.com/check/sendgrid/sendgrid-nodejs?pullRequest=547) it.</sub>

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+- [ ] Code passes all existing [tests](https://github.com/sendgrid/open-source-library-data-collector/tree/master/test)
+- [ ] Any new functionality added includes new unit tests in [`tests/test.py`](https://github.com/sendgrid/open-source-library-data-collector/blob/master/test/test.py)
+- [ ] Create or update example code to show the new functionality in action.
+- [ ] All code, branch, and git naming and style conventions are followed (see [`CONTRIBUTING.md`](https://github.com/sendgrid/open-source-library-data-collector/blob/master/CONTRIBUTING.md#style-guidelines--naming-conventions))
+- [ ] Feature branch has been rebased off of the latest `master` branch. ( see [`CONTRIBUTING.md`](https://github.com/sendgrid/open-source-library-data-collector/blob/master/CONTRIBUTING.md#creating-a-pull-request) ).
+
+If you have questions, please send an email [Sendgrid](mailto:dx@sendgrid.com), or file a Github Issue in this repository.
+
+<!--
```
Comment (mbernier) [inline .github/PULL_REQUEST_TEMPLATE.md]: Please remove this comment. Sorry about that.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+**Reason for the change**:
+
+**Link to original source**:
+
+### Checklist
+
+Make sure all of these items are complete, or else the PR will be ineligible for a code review.
+
+- [ ] Code passes all existing [tests](https://github.com/sendgrid/open-source-library-data-collector/tree/master/test)
```
Comment (mbernier) [inline .github/PULL_REQUEST_TEMPLATE.md]: Can you please change all instances of `open-source-library-data-collector` to `sendgrid-nodejs` ?

**Comment by shivakaushal** [comment]
@mbernier I've updated the template

**Comment by mbernier** [comment]
Thank you! 

Did you build reviewable? That thing looks pretty cool - I would love to learn more, please send me an email - my github username at sendgrid.com

**Comment by thinkingserious** [comment]
Hello @shivakaushal,<br /><br/>Thanks again for the PR!<br />                                              <br />It's <a href="https://sendgrid.com/blog/announcing-hacktoberfest-2017">HACKTOBERFEST</a>! We want to show our appreciation by sending                                              you some special Hacktoberfest swag. If you have not already, could you please fill out                                              <a href="https://dx.sendgrid.com/swag">this form</a> so we can send it to you? Thanks!                                             <br /><br />Team SendGrid DX

**Pull request merged**